QuizWhois/Common/Models/Base/BaseModel.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Common.Models.Base
+{
+    public abstract class BaseModel
+    {
+        public Guid Id { get; set; }
```

Comment:
Lets discourse using long for ids

Reply:
Seems reasonable since we have relatively small amount of objects in db and auto-incremented id could provide us some information compared to plain guid